docs/user/editor/calendars.md

```diff
@@ -19,7 +19,9 @@ Choose a calendar from the list to begin editing. To create a new calendar, clic
 
 Schedule exceptions allow users to define days where special services should replace the regularly operating calendars. To create a schedule exception, click the `Exceptions` tab and then click `+ New exception` (replaces the "New calendar" button).
 
-![schedule exception](../../img/schedule-exception.png)
+<div class="img-center">
+  ![create exception](../../gif/schedule-exception.gif)
```

Comment:
Broken link:

![image](https://user-images.githubusercontent.com/56846598/89803760-4195ad80-db01-11ea-848a-59f1e859d070.png)

Reply:
Oh I see. Remove the `<div>` if you want to use markdown syntax. If you need to use `<div>`, then you should use the HTML `<img>` tag inside.

Reply:
Another oddity: the correct path for the image seems to be `../../../gif/schedule-exception.gif`.

Reply:
^ ^ Still need to fix the URL @philip-cline!